setup.py

```diff
@@ -195,7 +196,7 @@ def __str__(self):
           ],
           'test': TESTS_REQUIRE,
           'docs': [
-              'Sphinx',
+              'Sphinx < 2',  # Version 2.0+ only supports Python 3.5+
```

Comment:
So what?  `tox -e docs` uses `basepython = python3.7`.  Let people who want to depend on `zope.security[docs]` on Python 2 do the `Sphinx < 2` thing themselves.

Reply:
That's a good idea, although there are Sphinx tests run in the test environments, see 
https://github.com/zopefoundation/zope.security/blob/fcc4a7f304c31cdb103a18ff02df8fc4d90f2b73/tox.ini#L8-L11

Reply:
Ah, I missed that bit!

[testenv:py27] deps = {[testenv]deps} and Sphinx < 2 ?